cmd/snap/cmd_find.go

```diff
@@ -86,20 +86,20 @@ func hasPrices(snaps []*client.Snap) bool {
 }
 
 func printWithPrices(w *tabwriter.Writer, snaps []*client.Snap, suggestedCurrency string) {
-	fmt.Fprintln(w, i18n.G("Name\tVersion\tPrice\tSummary"))
+	fmt.Fprintln(w, i18n.G("Name\tVersion\tStatus\tPrice\tSummary"))
```

Comment:
Let's please not show another column just for purchased snaps. We can say "(paid)" on the price column when that's the case.